packages/ui-extensions/src/surfaces/checkout/components/BlockLayout/examples/basic-blockLayout.example.ts

```diff
@@ -1,4 +1,4 @@
-import {extend, BlockLayout, View} from '@shopify/ui-extensions/checkout';
+import {extension, BlockLayout, View} from '@shopify/ui-extensions/checkout';
```

Comment:
The runtime code for all these examples are all still using `extend()` (sorry that I forgot about these)

Reply:
Maybe just remove the checkout stuff for now and we can fix it separately?

Reply:
Ok, let me remove them. I think I didn't see that all the examples for Checkout were out of date